kolibri_explore_plugin/assets/src/kolibriApi.js

```diff
@@ -21,6 +22,9 @@ const DEFAULT_HIDE_UNAVAILABLE = false;
 class KolibriApi {
   constructor(channelId) {
     this.channelId = channelId;
+
+    // Cache of template-ui translators:
+    this.translators = {};
```

Comment:
Also this should be a static class field. But the current linter is not happy about it. Has the linter regressed?